be/src/storage/aggregate_func.h

```diff
@@ -226,18 +226,11 @@ struct AggregateFuncTraits<OLAP_FIELD_AGGREGATION_MIN, OLAP_FIELD_TYPE_LARGEINT>
         if (src_null) return;
 
         bool dst_null = dst->is_null();
-        if (dst_null) {
-            dst->set_is_null(false);
-            memcpy(dst->mutable_cell_ptr(), src.cell_ptr(), sizeof(CppType));
-            return;
-        }
-
-        CppType dst_val, src_val;
-        memcpy(&dst_val, dst->cell_ptr(), sizeof(CppType));
-        memcpy(&src_val, src.cell_ptr(), sizeof(CppType));
-        if (src_val < dst_val) {
-            dst->set_is_null(false);
-            memcpy(dst->mutable_cell_ptr(), src.cell_ptr(), sizeof(CppType));
+        CppType* dst_val = reinterpret_cast<CppType*>(dst->mutable_cell_ptr());
```

Comment:
These cell addresses are not memory aligned and the compiler will generate SIMD instructions when optimising, which will cause crash when the addresses are not aligned.
So here memcopy is used to ensure that the program is correct.

Reply:
Some comments should be added here, otherwise no one knows why it is written like this.

Reply:
https://github.com/StarRocks/starrocks/blob/main/be/src/util/unaligned_access.h